doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -138,28 +137,30 @@ objects.
     cephfs-data-scan init
 
 Finally, you can regenerate metadata objects for missing files
-and directories based on the contents of a data pool.  This is
-a three-phase process.  First, scanning *all* objects to calculate
-size and mtime metadata for inodes.  Second, scanning the first
-object from every file to collect this metadata and inject it into
-the metadata pool. Third, checking inode linkages and fixing found
-errors.
+and directories based on the contents of a data pool. This is
+a three-phase process: 
+
+#. Scanning *all* objects to calculate size and mtime metadata for inodes.  
+#. Scanning the first object from every file to collect this metadata and
+   inject it into the metadata pool. 
+#. Checking inode linkages and fixing found errors.
 
 ::
 
     cephfs-data-scan scan_extents [<data pool> [<extra data pool> ...]]
     cephfs-data-scan scan_inodes [<data pool>]
     cephfs-data-scan scan_links
 
-'scan_extents' and 'scan_inodes' commands may take a *very long* time
-if there are many files or very large files in the data pool.
+``scan_extents`` and ``scan_inodes`` commands may take a *very long* time if
+the data pool contains many files or very large files.
 
-To accelerate the process, run multiple instances of the tool.
+To accelerate the process of running ``scan_extents`` or ``scan_inodes``, run
+multiple instances of the tool.
 
 Decide on a number of workers, and pass each worker a number within
-the range 0-(worker_m - 1).
+the range ``0-(worker_m - 1)`` (that is, 'zero to "worker_m" minus 1').
```

Comment:
Not clear what `worker_n` and `worker_m` are here.  Literals? I suspect `worker_m` is the total number of workers and `worker_n` is ordinal?

Reply:
We can hash this out when we get to the part of this project where we update the technical content.